go/minterm/minterm_nix.go

```diff
@@ -26,3 +26,7 @@ func (m *MinTerm) open() error {
 	m.width, m.height = w, h
 	return nil
 }
+
+func (m *MinTerm) getReadWriter() io.ReadWriter {
```

Comment:
Whoops, thought I could get away with it. Would never merge with such a CI failure anywyaze.